test/css/value/unicode.0.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "value",
+  "content": [
+    {
+      "type": "ident",
+      "content": "U",
```

Comment:
> what about urange or unicode-range ?

Oh, what a cool spec you've found! As I get it, `unicode-range` may be a comma-separated list of `urange` and if we want to only define `U+` thing, it should be `urange`.